app_book/models.py

```diff
@@ -17,6 +17,7 @@ def book_directory_path(instance, filename):
 class Book(models.Model):
     user = models.ForeignKey(User, default=None)
 
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
Mit auto_now=ture wird der timpestamp jedes mal erneuert wen das Objekt gespeichert wird, mit auto_now_add=ture wird ein fester timpestamp beim erstellen des Objekts gesetzt.

"As currently implemented, setting auto_now or auto_now_add to True will cause the field to have editable=False and blank=True set."